src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,50 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin Desktop {
+  @media (min-width: 1280px) {
+    @content;
+  }
+}
+  
+@mixin Tablet {
```

Comment:
It's a good practice to name mixins with lowercase and hyphens, like 'tablet' instead of 'Tablet'.